internal_worker_base.go

```diff
@@ -36,8 +36,8 @@ import (
 )
 
 const (
-	retryPollOperationInitialInterval    = time.Millisecond
-	retryPollOperationMaxInterval        = 1 * time.Second
+	retryPollOperationInitialInterval    = 20 * time.Millisecond
+	retryPollOperationMaxInterval        = 60 * time.Second
```

Comment:
Can we use 60 seconds as the expiration interval and 10 seconds as max interval?
We should also emit a log if poll fails after all retries are still exhausted.